### ISSUE TITLE
Installation not working - no framework provider

### ISSUE THREAD

**Issue by martinsson** [radify/karma-es6-shim#14]
I installed added the framework to karma.conf and tried to start the tests. 

```
npm install karma-es6-shim
karma-es6-shim@0.2.3 node_modules/karma-es6-shim
├── es5-shim@4.4.2
└── es6-shim@0.34.4

npm test  


~coolApp/node_modules/karma/node_modules/di/lib/injector.js:9
      throw error('No provider for "' + name + '"!');
            ^
Error: No provider for "framework:es6-shim"! (Resolving: framework:es6-shim)
    at error (~coolApp/node_modules/karma/node_modules/di/lib/injector.js:22:68)
```

Sure I don't use requirejs but I'm guessing it has nothing to do with the error.
I too have npm 2.x so I guess I'm also subject to #13 but I don't seem to have the same error.

**Comment by gavD** [comment]
I'll take a look at this and #13, probably on Monday. Thanks for reporting
it!

**Comment by gavD** [comment]
Hi,

I have fixed #13 and updated the [sample project to 1.0.0](https://github.com/radify/karma-es6-shim-example). If you [look at the changes I made](https://github.com/radify/karma-es6-shim-example/pull/6/files) you should be able to resolve this issue. I think explicitly loading the plugins in karma.conf.js should be what you need to do once you update to v1.0.0 of karma-es6-shim.

Thanks,
- Gavin
